scripts/README.md

```diff
@@ -0,0 +1,175 @@
+
+```sh
+npx truffle exec build/common/scripts/get_auction_elements.js --network rinkeby --expired true --covered false --tokens 0,1,2,3,4 --pageSize 50
+```
+
+Be warned that this may take a while!
+
+### Transitive Orderbook
+
+Computes a price at which the given amount of `sellToken` can be sold for `buyToken` given the orderbook of the batch that is currently collecting orders.
+
+```sh
+npx truffle exec build/common/scripts/transitive_orderbook.js --network rinkeby --sellToken 1 --buyToken 4 --sellAmount 1000
+```
+
+with additional optional agruments such as `hops` and `pageSize`
+
+## Involved Transactions
+
+### Add Token List
+
+This script is usually only needed upon initial deployment of a new BatchExchange contract to any network. It will then fetch a currated list of ERC20 tokens and register each of these for excahnge on the newly deployed contract.
+
+```sh
+npx truffle exec build/common/scripts/add_token_list.js
+```
+
+### Ensure OWL Liquidity
+
+This is meant to check and ensure that there are orders selling OWL at an appropriate price between all listed token on the exchange.
+
+```sh
+npx truffle exec build/common/scripts/ensure_owl_liquidity.js --network rinkeby
+```
+
+### Place Spread Orders
+
+This script is meant to be used with a token list of only stable tokens that are all valued essentially equally. This will place orders with configurable spread between each token pair listed. This script will inform the user of the orders that about to be placed and between which tokens and end with a user prompt confirming that the order placement transaction should be sent.
+
+Note that any undesired order placements can be cancelled using the `cancel_order` script along with the order Ids logged as a result of this script's execution. Furthermore, when default values for `validFrom` are used, the orders will not become valid for three batches (to allow ampel time for cancellation).
+
+```sh
+npx truffle exec build/common/scripts/place_spread_orders.js  --tokens 2,3,6 --network rinkeby
+```
+
+## Development Interactions
+
+These scripts serve as easy-access development tooling on ganache. Some of the simplest two are `wait_seconds` and `close_auction` which do exactly this. In fact, `close_auction` is just the combination of `wait_seconds` with the number of seconds remaining in the current batch (acquired via `invoke_view_function` as in the example above).
+
+None of the development scripts will work on any live ethereum networks since they all rely on two features exclusive to ganache (i.e. `evm_mine` and `evm_increaseTime`) do not exhibit the ability to increase time, or trivially mine blocks.
+
+Before executing any of these scripts, please ensure you have a running instance of ganache (with a gas limit of at least 8e6) and that the appropriate contracts have been deployed to your local network
+
+```sh
+ganche-cli -l 8e6
```

Comment:
Maybe suggest using `yarn ganache` instead, this runs Ganache with the correct parameters.